### PR TITLE
Clarify the results of `VectorN.normalized()` in the docs

### PR DESCRIPTION
**Pull request by GrahameGW**
The current documentation for `VectorN.normalized()` does not explicitly state what happens if you normalize a zero-length vector. As there are at least two logical outcomes (a divide by zero error or `VectorN.Zero`), it is worth it to specify how godot handles this case; something which is potentially further confused by the note RE potential odd behavior when the vector length is very small.

This change simply adds the clarifying note that normalizing a zero-length vector will return `(0,0)`, `(0,0,0)`, or `(0,0,0,0)`, depending on the vector size normalized.

### CODE REVIEW

File: doc/classes/Vector2.xml [hunk 1]
```diff
@@ -324,7 +324,7 @@
 		<method name="normalized" qualifiers="const">
 			<return type="Vector2" />
 			<description>
-				Returns the result of scaling the vector to unit length. Equivalent to [code]v / v.length()[/code]. See also [method is_normalized].
+				Returns the result of scaling the vector to unit length.  Equivalent to [code]v / v.length()[/code]. Returns [code](0,0)[/code] if [code]v.length() == 0[/code]. See also [method is_normalized].
```
Comment (RedMser) [inline doc/classes/Vector2.xml:327]:
```suggestion
				Returns the result of scaling the vector to unit length. Equivalent to [code]v / v.length()[/code]. Returns [code](0,0)[/code] if [code]v.length() == 0[/code]. See also [method is_normalized].
```

You added an extra space here

File: doc/classes/Vector3.xml [hunk 2]
```diff
@@ -293,7 +293,7 @@
 		<method name="normalized" qualifiers="const">
 			<return type="Vector3" />
 			<description>
-				Returns the result of scaling the vector to unit length. Equivalent to [code]v / v.length()[/code]. See also [method is_normalized].
+				Returns the result of scaling the vector to unit length. Equivalent to [code]v / v.length()[/code]. Returns [code](0,0,0)[/code] if [code]v.length() == 0[/code]. See also [method is_normalized].
```
Comment (AThousandShips) [inline doc/classes/Vector3.xml:296]:
```suggestion
				Returns the result of scaling the vector to unit length. Equivalent to [code]v / v.length()[/code]. Returns [code](0, 0, 0)[/code] if [code]v.length() == 0[/code]. See also [method is_normalized].
```

File: doc/classes/Vector4.xml [hunk 3]
```diff
@@ -227,7 +227,7 @@
 		<method name="normalized" qualifiers="const">
 			<return type="Vector4" />
 			<description>
-				Returns the result of scaling the vector to unit length. Equivalent to [code]v / v.length()[/code]. See also [method is_normalized].
+				Returns the result of scaling the vector to unit length. Equivalent to [code]v / v.length()[/code]. Returns [code](0,0,0,0)[/code] if [code]v.length() == 0[/code]. See also [method is_normalized].
```
Comment (AThousandShips) [inline doc/classes/Vector4.xml:230]:
```suggestion
				Returns the result of scaling the vector to unit length. Equivalent to [code]v / v.length()[/code]. Returns [code](0, 0, 0, 0)[/code] if [code]v.length() == 0[/code]. See also [method is_normalized].
```

**Review by AThousandShips — APPROVED** [review]

**Comment by GrahameGW** [comment]
Fixed space typo and adjusted vector format to (0, 0) from (0,0)-- thanks guys for the review

**Comment by AThousandShips** [comment]
Could you please update the commit message to match the PR title?

**Comment by GrahameGW** [comment]
> Could you please update the commit message to match the PR title?

Yep, all set

**Comment by akien-mga** [comment]
Thanks! And congrats for your first merged Godot contribution :tada:

**Pull request merged**